site/gatsby-site/src/templates/entity.js

```diff
@@ -163,7 +163,11 @@ const EntityPage = ({ pageContext, data, ...props }) => {
       addToast({
         message: (
           <Trans i18n={i18n}>
-            Please <Link to={localizePath({ path: '/login' })}>log in</Link> to subscribe
+            Please{' '}
+            <Link to={localizePath({ path: `/login?redirectTo=${props?.location?.pathname}` })}>
+              log in
+            </Link>{' '}
+            to subscribe
```

Comment:
The same here. Please fix the translations. They are broken after this change with `{' '}`

Spanish version:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/6564809/224134948-510cf04d-d3a7-460a-8e2b-eb3e9647f956.png">

Reply:
@clari182 Please go to https://deploy-preview-1713--staging-aiid.netlify.app/es/entities/facebook/ and click on `Notificarme de nuevos Incidentes de Facebook` and you will see the toast in English